pinax/stripe/models.py

```diff
@@ -132,10 +168,13 @@ class Customer(StripeObject):
 
     @property
     def stripe_customer(self):
-        return stripe.Customer.retrieve(self.stripe_id)
+        return stripe.Customer.retrieve(
+            self.stripe_id,
+            stripe_account=self.stripe_account
+        )
 
     def __str__(self):
-        return str(self.user)
+        return unicode(self.user)
```

Comment:
shouldn't need `unicode` here because of the `@python_2_unicode_compatible` class decorator.